packs/classes/bard.json

```diff
@@ -175,30 +175,7 @@
                 "key": "ActiveEffectLike",
                 "mode": "upgrade",
                 "path": "system.proficiencies.spellcasting.rank",
```

Comment:
These should move to their respective spellcasting features. Newer ones are "Bard Spellcasting" while older ones are "Arcane Spellcasting" or some such.

Reply:
I believe those are the archetype feats.

Reply:
Or do you mean "Wizard Spellcasting", "Spell Repertoire (Bard)", etc?

Reply:
The initial training does come from the class rather than any particular feature, but we can make a spot for it in the data.